release-notes/2023-07/SystemsConnectionMigration.md

```diff
@@ -0,0 +1,38 @@
+# **Migration plan for systems connectivity that is affected by the breaking change in the salt-master chart.**
+
+## The current release is introducing a breaking change regarding the systems connectivity. This means that any SLE <=0.14 release is affected by this change. The saltmaster service which manages the connectivity with the client systems has been replaced with a new implementation which causes all the connected clients to become disconnected. The next steps will describe how to re-approve and re-connect all the previously connected clients
+
+&nbsp;
+
+## **Before doing the upgrade of SLE**
+
+The old salt-master has a pair of RSA keys that are used for securing the communication. The public key is already cached on all the
+accepted clients. We need to persist these keys between all the SLE versions, otherwise the client systems will reject the new salt-master
+that would have a different public key.
+
+1. Open a terminal to the current "saltmaster-0" pod.
```

Comment:
Generally, ordered lists in markdown can use `1.` for all items. This makes it lower churn if you need to reorder or insert new items in the list. e.g. 

```
1. foo
1. bar
1. bam
```

Is rendered as:
1. foo
1. bar
1. bam